CHANGELOG.md

```diff
@@ -2,67 +2,9 @@
 
 Please see https://github.com/rapidsai/cugraph/releases/tag/v22.04.00a for the latest changes to this development branch.
 
-# cuGraph 22.02.00 (2 Feb 2022)
+# cuGraph 22.02.00 (Date TBD)
```

Comment:
These changelog changes are invalid/outdated. Can you merge the latest changes from `branch-22.04` to your PR?

Reply:
I found that merge attempts from `branch-22.04` would not work, so my solution was to make another branch directly from `branch-22.04`, called `branch-22.04-fea-node2vec-pylib`, and created a new PR #2084, rendering this one obsolete.